spec/argv_spec.rb

```diff
@@ -53,6 +53,13 @@ module CLAide
         @argv.remainder.should == %w(--flag ARG1 ARG2 --no-other-flag)
       end
 
+      it 'allows mutliple values for the same option' do
+        @argv = ARGV.new %w(--ignore=foo --ignore=bar)
+        @argv.option('ignore').should == 'bar'
+        @argv.option('ignore').should == 'foo'
+        @argv.option('ignore').should.be.nil
+      end
```

Comment:
I'm switching Xcodeproj to CLAide, and I needed a way to allow ignoring multiple things, and I wanted to do it in the most minimally-invasive way possible. I could add a new help to return all options for a given value?